### PR TITLE
Specification files for DinoV2 model versions for image embedding

### PR DESCRIPTION
**Pull request by rdondera-microsoft**
Two model versions: base and giant.

Models in registry:
https://ml.azure.com/registries/automl-images-dev/models/Facebook-DinoV2-Image-Embeddings-ViT-Base/version/3?tid=72f988bf-86f1-41af-91ab-2d7cd011db47#overview
https://ml.azure.com/registries/automl-images-dev/models/Facebook-DinoV2-Image-Embeddings-ViT-Giant/version/1?tid=72f988bf-86f1-41af-91ab-2d7cd011db47#overview

Model publishing pipeline runs:
https://ml.azure.com/experiments/id/d1873606-5a43-4303-9053-6fb512612c9f/runs/heroic_gold_0dg8t820hd?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourceGroups/phmantri_eastus/providers/Microsoft.MachineLearningServices/workspaces/phmantri_eastus_2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#
https://ml.azure.com/experiments/id/d1873606-5a43-4303-9053-6fb512612c9f/runs/nice_door_gzpr3vfqx0?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourceGroups/phmantri_eastus/providers/Microsoft.MachineLearningServices/workspaces/phmantri_eastus_2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

### CODE REVIEW

File: assets/models/system/facebook-dinov2-image-embeddings-base/model.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+path:
+  container_name: finetuning-image-models
+  container_path: HuggingFace-mlflow/Facebook-DinoV2-Image-Embeddings-ViT-Base/1706570543/mlflow_model_folder
+  storage_name: automlcesdkdataresources
+  type: azureblob
+publish:
+  description: description.md
+  type: mlflow_model
```
Comment (novaturient95) [inline assets/models/system/facebook-dinov2-image-embeddings-base/model.yaml:8]: Nit: Add a new line

Comment (novaturient95) [inline assets/models/system/facebook-dinov2-image-embeddings-base/model.yaml:8]: Nit: Add a new line

File: assets/models/system/facebook-dinov2-image-embeddings-base/description.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+|--|--|--|
+Real time|<a href=\"https://aka.ms/azureml-infer-sdk-image-embeddings\" target=\"_blank\">image-embeddings-online-endpoint.ipynb</a>|<a href=\"https://aka.ms/azureml-infer-cli-image-embeddings\" target=\"_blank\">image-embeddings-online-endpoint.sh</a>
+
+### Sample inputs and outputs (for real-time inference)
+
+#### Sample input
+
+```json
+{
+   \"input_data\":{
```
Comment (novaturient95) [inline assets/models/system/facebook-dinov2-image-embeddings-base/description.md:24]: nit: is escape needed in sample inputs and outputs?

File: assets/models/system/facebook-dinov2-image-embeddings-base/spec.yaml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+            Standard_NC12s_v3,
+            Standard_NC16as_T4_v3, 
+            Standard_NC24s_v3, 
+            Standard_NC64as_T4_v3, 
+            Standard_NC24ads_A100_v4,
+            Standard_NC48ads_A100_v4, 
+            Standard_NC96ads_A100_v4, 
+            Standard_ND96asr_v4, 
+            Standard_ND96amsr_A100_v4,
+            Standard_ND40rs_v2
```
Comment (novaturient95) [inline assets/models/system/facebook-dinov2-image-embeddings-base/spec.yaml:79]: Nit: add a new line

File: assets/models/system/facebook-dinov2-image-embeddings-base/description.md (hunk 2 above)
Comment (rdondera-microsoft) [reply assets/models/system/facebook-dinov2-image-embeddings-base/description.md:24]: Definitely not, thank you for catching.

File: assets/models/system/facebook-dinov2-image-embeddings-base/model.yaml (hunk 1 above)
Comment (rdondera-microsoft) [reply assets/models/system/facebook-dinov2-image-embeddings-base/model.yaml:8]: Done.

Comment (rdondera-microsoft) [reply assets/models/system/facebook-dinov2-image-embeddings-base/model.yaml:8]: Done.

File: assets/models/system/facebook-dinov2-image-embeddings-base/spec.yaml (hunk 3 above)
Comment (rdondera-microsoft) [reply assets/models/system/facebook-dinov2-image-embeddings-base/spec.yaml:79]: Done.

File: assets/models/system/facebook-dinov2-image-embeddings-base/description.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+The Vision Transformer (ViT) is a transformer encoder model (BERT-like) pretrained on a large collection of images in a self-supervised fashion.
```
Comment (gauravrajguru) [inline assets/models/system/facebook-dinov2-image-embeddings-base/description.md:1]: nit: it sounds more like vit model description. It doesn't include keyword `dino`. Could you please check.

File: assets/models/system/facebook-dinov2-image-embeddings-base/description.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+Images are presented to the model as a sequence of fixed-size patches, which are linearly embedded. One also adds a [CLS] token to the beginning of a sequence to use it for classification tasks. One also adds absolute position embeddings before feeding the sequence to the layers of the Transformer encoder.
+
+Note that this model does not include any fine-tuned heads.
+
+By pre-training the model, it learns an inner representation of images that can then be used to extract features useful for downstream tasks: if you have a dataset of labeled images for instance, you can train a standard classifier by placing a linear layer on top of the pre-trained encoder. One typically places a linear layer on top of the [CLS] token, as the last hidden state of this token can be seen as a representation of an entire image.
+
+### Intended uses & limitations
```
Comment (gauravrajguru) [inline assets/models/system/facebook-dinov2-image-embeddings-base/description.md:9]: FYI, we need to follow this specification for writing model description https://microsoftapc-my.sharepoint.com/:w:/r/personal/ayushmishra_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B2E13DB38-A1A9-4148-B1DC-AC14A76E1F72%7D&file=System%20Registry%20model%20card%20alignment.docx&action=default&mobileredirect=true&DefaultItemOpen=1&share=IQE42xMuqaFIQbHcrBSnbh9yAamNOJxrsZ9DvamUluUGp40

@novaturient95 please correct me if anything changed?

**Review by gauravrajguru — DISMISSED** [review]
minor comments.

File: assets/models/system/facebook-dinov2-image-embeddings-base/description.md (hunk 4 above)
Comment (rdondera-microsoft) [reply assets/models/system/facebook-dinov2-image-embeddings-base/description.md:1]: This _is_ a ViT model; DinoV2 is a method to train transformer models.

Will include "DinoV2" so we eliminate any kind of potential confusion.

File: assets/models/system/facebook-dinov2-image-embeddings-base/description.md (hunk 5 above)
Comment (rdondera-microsoft) [reply assets/models/system/facebook-dinov2-image-embeddings-base/description.md]: Done.

**Review by novaturient95 — APPROVED** [review]

**Pull request merged**